src/components/calcite-tooltip-manager/calcite-tooltip-manager.tsx

```diff
@@ -0,0 +1,96 @@
+import { Component, Element, Host, h, Prop } from "@stencil/core";
+import { TOOLTIP_REFERENCE } from "../calcite-tooltip/resources";
+
+@Component({
+  tag: "calcite-tooltip-manager"
+})
+export class CalciteTooltipManager {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * CSS Selector to match reference elements for tooltips.
+   */
+  @Prop() selector = `[${TOOLTIP_REFERENCE}]`;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteTooltipManagerElement;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  // --------------------------------------------------------------------------
+
+  componentDidLoad() {
+    const { el } = this;
+
+    el.addEventListener("mouseenter", this.show, true);
```

Comment:
Just an observation, but there's a lot in common with the popover manager here. Could the common stuff be extracted where the concrete managers would just need to register the events and handlers?

Reply:
yeah, I could share some code. 👍